sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProtoTest.java

```diff
@@ -269,6 +269,7 @@ enum TestEnum {
           .endRecord();
 
   private static final Schema SCHEMA_WITH_MAP;
+  private static final Schema SCHEMA_WITH_NULLABLE_ARRAY;
```

Comment:
Out of curiosity, is there a benefit to declaring these variables here but initializing them later in a `static { }` block? Asking because we conventionally initialize them in the same line.

Reply:
I don't think so, it must be something with refactoring I did. correcting them.